crds/zz_defs.go

```diff
@@ -98,7 +98,7 @@ func operatorsCoreosCom_catalogsourcesYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "operators.coreos.com_catalogsources.yaml", size: 7231, mode: os.FileMode(420), modTime: time.Unix(1591650274, 0)}
+	info := bindataFileInfo{name: "operators.coreos.com_catalogsources.yaml", size: 7231, mode: os.FileMode(436), modTime: time.Unix(1593116031, 0)}
```

Comment:
this changed as a result of running codegen? A little confused as to why file permissions would change as a result of adding a comment to the source code.

Reply:
yeah, as a result of `make generate`. I don't really understand this, I would not have expected this change but instead a change of the generated openapi schemes in `crds/operators.coreos.com_installplans.yaml` and `crds/operators.coreos.com_subscriptions.yaml`. To me, it looks like either I am using this somehow wrong or its broken. I was hoping there would be a job that fails and tells me what to do but doesn't seem to be the case :upside_down_face:

Reply:
This is just a bit strange. Maybe you can discard these changes from the PR and just keep the main change in `pkg/operators/installplan_types.go`. Then I can lgtm and merge this PR. Thanks.

Reply:
@dinhxuanvu updated, PTAL